Lib/fontbakery/profiles/googlefonts.py

```diff
@@ -1177,9 +1177,11 @@ def com_google_fonts_check_name_unwanted_chars(ttFont):
 
 @check(
     id = 'com.google.fonts/check/usweightclass',
-    conditions = ['expected_style'],
+    conditions = ['expected_font_names'],
     rationale = """
-        Google Fonts expects variable fonts, static ttfs and static otfs to have
+        Check the OS/2 usWeightClass is appropriate for the font's best SubFamily name.
```

Comment:
Some reporters, like the terminal one, will display both the check description (the actual check function doc-string) and the rationale text. So this first line will end up displayed twice because it is also in the check description.